src/components/EventsForm.tsx

```diff
@@ -0,0 +1,210 @@
+  initialValues?: EventsFormInitialValues;
+  edit: boolean;
+  eventRequest: AsyncRequest<EventInformation, any>;
+}
+
+const EventsForm: React.FC<EventsFormProps> = ({
+  onFinish,
+  initialValues,
+  edit,
+  eventRequest,
+}) => {
+  return (
+    <FormContainer>
+      <Helmet>
+        <title>Create Event</title>
+      </Helmet>
+      <Form
+        name="basic"
+        layout="vertical"
+        initialValues={initialValues}
+        onFinish={onFinish}
+      >
+        <Form.Item
+          label="Event Name"
+          name="title"
+          rules={[
+            { required: true, message: 'Please input the name of the event' },
+          ]}
+        >
+          <Input placeholder="Event Name" />
+        </Form.Item>
+
+        <Form.Item
+          label="Event Location"
+          name="location"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the location of the event',
+            },
+          ]}
+        >
+          <Input placeholder="Event Location" />
+        </Form.Item>
+
+        <InlineFormItem
+          name="start"
+          fieldKey="start"
+          label="Start Date"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the start date and time of the event',
+            },
+          ]}
+        >
+          <DatePicker showTime use12Hours format="MM/DD/YYYY hh:mm A" />
+        </InlineFormItem>
+
+        <DateText>to</DateText>
+
+        <InlineFormItem
+          name="end"
+          fieldKey="end"
+          label="End Date"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the end date and time of the event',
+            },
+          ]}
+        >
+          <DatePicker showTime use12Hours format="MM/DD/YYYY hh:mm A" />
+        </InlineFormItem>
+
+        <Form.Item
+          label="Spots Available"
+          name="capacity"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the number of spots available',
+            },
+          ]}
+        >
+          <HalfWidthInput placeholder="Spots Available" min={0} />
+        </Form.Item>
+
+        <Form.Item
+          label="Price"
+          name="price"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the price of the event',
+            },
+          ]}
+        >
+          <HalfWidthInput defaultValue={0} min={0} />
+        </Form.Item>
+
+        <Form.Item
+          label="Description"
+          name="description"
+          rules={[
+            {
+              required: true,
+              message: 'Please input an event description',
+            },
+          ]}
+        >
+          <TextArea rows={3} placeholder="Description" />
+        </Form.Item>
+
+        {/* We will come back to this in a second PR because it requires backend changes too :)
```

Comment:
```suggestion
        {/* TODO: come back to this in a second PR because it requires backend changes too :)
```
So IDEs can recognize